### PR TITLE
Fix for non staff user view/change business info modal

### PR DESCRIPTION
**Pull request by BrandonSharratt**
*Issue:*https://github.com/bcgov/entity/issues/24459

*Description of changes:*
Fix the text that shows for non staff user.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Could you please trigger the preview site build and/or provide before and after screenshots to show what this change did? Thanks.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-84-j8v0hdv5.web.app

**Comment by severinbeauvais** [comment]
My test results...

## Alteration

### As a regular user

image

### As staff

I am redirected to Edit UI to perform the alteration. This is consistent with Filings UI.

## Voluntary Dissolution

### As a regular user

**This is not consistent with Filings UI, which shows you the action in the More Actions dropdown, and then a dialog when you select it.**

image

### As staff

image

**Review by severinbeauvais — COMMENTED** [review]
If you don't have a UI design to be different from Filings UI then please update this.

Also, please use a more verbose PR title :)

**Comment by BrandonSharratt** [comment]
This PR is not addressing visibility or enabled state of actions.

File: src/components/bcros/businessDetails/Links.vue [hunk 1]
```diff
@@ -63,7 +63,7 @@ const setShowDissolutionDialog = (show: boolean) => {
 }
 
 const dialogTitle = computed<string>(() => {
-  return (!currentBusiness?.value?.goodStanding && hasRoleStaff)
+  return showChangeNotInGoodStandingDialog.value
```
Comment (severinbeauvais) [inline src/components/bcros/businessDetails/Links.vue:66]: I don't understand how a variable for whether to show a dialog can be used to select one other other string (or template, below).

I cannot verify this as I cannot get to the dissolution version of this dialog.

**Comment by severinbeauvais** [comment]
> However, in the case demonstrated it should be disabled. The business is not in good standing, so it does not show in the allowable actions for this business which is considered a blocker in the legal api.

So... the dissolution version of this dialog will NEVER show?

**Comment by BrandonSharratt** [comment]
> > However, in the case demonstrated it should be disabled. The business is not in good standing, so it does not show in the allowable actions for this business which is considered a blocker in the legal api.
> 
> So... the dissolution version of this dialog will NEVER show?

No it'll show you just need a business in good standing

**Comment by hfekete** [comment]
ticket to verify dissolve flow for not in good standing business:
https://app.zenhub.com/workspaces/beneficial-ownership-2023-6557a7db7ce6e464af821855/issues/gh/bcgov/entity/24637

**Comment by severinbeauvais** [comment]
> > > However, in the case demonstrated it should be disabled. The business is not in good standing, so it does not show in the allowable actions for this business which is considered a blocker in the legal api.
> > 
> > 
> > So... the dissolution version of this dialog will NEVER show?
> 
> No it'll show you just need a business in good standing

Is there still a "Business is not in good standing -- voluntary dissolution" dialog? How do I get to it?

**Review by severinbeauvais — APPROVED** [review]
> Lets close this PR if it fixes what the ticket asks for, and lets create new verify or bug ticket if we are having some issues in a flow.

Go for it.

**Pull request merged**